README.md

```diff
@@ -27,12 +43,32 @@ Ask us questions, talk about contributing, or just say hi on [Slack](https://vsc
 
 Donations help convince me to work on this project rather than my other (non-open-source) projects. I'd love to work on VSCodeVim full time, but I need money to live!
 
+## Contents
 
-## Configuring VSCodeVim
+* [Quick example settings](#quick-example-settings)
```

Comment:
I think `quick example settings` would below under the `settings` category.

Mac/Windows setup could be under it's own `setup`/getting started category.